resources/benchmark/tpcds/tables/customer.csv.json

```diff
@@ -98,6 +98,7 @@
     "quote": "\"",
     "reject_quoted_nonstrings": true,
     "reject_null_strings": false,
+    "unquoted_null_string_is_string": true,
```

Comment:
Using an enum right? I tried to do that, but gave up after a while; parsing an enum from json is not as easy as it might seem and I decided it is not worth the effort. I can do it though if you think it is important enough.

Reply:
What is the problem there? I'd have placed a string in the JSON, added a bimap `enum class <=> string` and used that for the conversion. Am I missing something?

Reply:
Got it now. You are right, it was not that hard.